packages/thyra/core/src/interfaces/operator_vector/fundamental/Thyra_VectorSpaceBase_decl.hpp

```diff
@@ -709,6 +709,75 @@ class VectorSpaceBase : virtual public Teuchos::Describable {
   createMembersView(
     const RTOpPack::ConstSubMultiVectorView<Scalar> &raw_mv ) const = 0;
 
+
+ public:
+
+  /** \brief Create a (possibly) cached multi-vector member that is a non-<tt>const</tt> view of
+   * raw multi-vector data. The caching mechanism must be implemented by child classes, by default
+   * this just calls the regular <tt>createMembersView</tt>.
+   *
+   * @param raw_mv [in] On input contains pointer
+   * (i.e. <tt>raw_mv.values()</tt>) to array that the returned
+   * <tt>MultiVectorBase</tt> will be a view of.
+   *
+   * <b>Preconditions:</b><ul>
+   *
+   * <li><tt>raw_mv</tt> has been initialized to memory (i.e.
+   * <tt>raw_mv.subDim()!=0 && raw_mv.values()!=NULL</tt>).
+   *
+   * <li><tt>raw_mv</tt> is <em>consistent</em> with the local storage of this
+   * spaces vector data.  This precondition is purposefully vague since this
+   * function can be used an variety of specialized use-cases.
+   *
+   * </ul>
+   *
+   * <b>Postconditions:</b><ul>
+   *
+   * <li>See <tt>this->createMembers()</tt> where
+   * <tt>numMembers==raw_mv.numSubCols()</tt>
+   *
+   * </ul>
+   *
+   * It is stated here that the client can not expect that the values pointed
```

Comment:
You could just say that "the return value is a generalized view of `raw_mv`'s data."  Thyra's documentation explains what "generalized view" means.